roles/wazo-vars/defaults/main.yml

```diff
@@ -4,6 +4,9 @@ wazo_debian_repo: main
 wazo_debian_repo_upgrade: main
 wazo_distribution: wazo-dev-buster
 wazo_distribution_upgrade: wazo-dev-buster
+wazo_meta_package: wazo-platform
+# wazo_extra_repository: deb http://mirror.wazo.community/extra/ extra-buster main
+# wazo_extra_repository_filename: wazo-extra
```

Comment:
we could make it a default to allow the user to only configure wazo_extra_repository

Reply:
I'm not sure how to do this, because we need to have the wazo mirror AND the extra for our use case, not only one or the other

Reply:
I think Fred means to set a real default to the variable `wazo_extra_repository_filename`, so that someone only has to set only `wazo_extra_repository`, instead of both `wazo_extra_repository` AND `wazo_extra_repository_filename`

Reply:
Aah, so yes I can uncomment the line :)